includes/Admin/Settings.php

```diff
@@ -760,6 +760,13 @@ public function get_settings_fields() {
                         'address' => __( 'Store Address', 'dokan-lite' ),
                     ],
                 ],
+                'enable_dokan_fontawesome' => [
```

Comment:
settings name should be `disable_dokan_fontawesome` since this feature was enabled by default.